python/rascaline/clib.py

```diff
@@ -15,8 +16,36 @@ def __call__(self):
             path = _lib_path()
             self._cache = cdll.LoadLibrary(path)
             setup_functions(self._cache)
+            _set_default_logging_callback()
         return self._cache
 
+def _set_default_logging_callback():
+    '''
+    Default logging function. For now print is used.
+    '''
+    def default_print(level, string):
+        print(string)
```

Comment:
Instead of `print`, I think the default should use the [logging](https://docs.python.org/3/library/logging.html) module for better integration with the ecosystem

Reply:
Yes this was the plan, I thought this would be the next PR, but actually its very simple so I changed to logging